tests/contrib/database/test_mysql.py

```diff
@@ -5,16 +5,49 @@
 import sys
 import shutil
 import tempfile
+import six
+
+
+from charmhelpers.fetch import (
+    apt_install,
+    apt_update,
+    filter_installed_packages,
+)
```

Comment:
Please could you use a qualified import rather than bring the names into the module's namespace.  e.g.

```python
import charmhelpers.fetch as fetch

# the use it as
fetch.apt_install(...)
```

It's preferred to do this (generally) as it makes it obvious *where* a function lives.

Reply:
I simply copied lines 44-48 from `charmhelpers/contrib/database/mysql.py` file. Will re-patch.

Reply:
Ah, it's "old style" vs "new style" tensions! :)  Originally, in Python, the preferred way *was* to do "from xyz import abc".  However, more recently (especially in the openstack charms team) we've been pushing for the changed "import xyz" or "import a.b.c.d as e" and then using the name of the module (or its alias) in front of a foreign module function.  This is to help with maintenance really when reading code.  Thanks!